import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -0,0 +1,462 @@
+				Policy:   throttlePolicy,
+				Type:     applicationThrottlePolicyFlag,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := superAdminUser
+			adminPassword := superAdminPassword
+			if isTenantUser(args.CtlUser.Username, TENANT1) {
+				adminUsername = adminUsername + "@" + TENANT1
+			}
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.ApplicationThrottlePolicyType)
+		})
+	}
+}
+
+// Import an already existing Advanced Throttling Policy to the destination env without update
+func TestImportUpdateConflictAdvancedThrottlePolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, dev, user.Admin.Username, user.Admin.Password, apim.AdvancedThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     advancedThrottlePolicyFlag,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := superAdminUser
+			adminPassword := superAdminPassword
+			if isTenantUser(args.CtlUser.Username, TENANT1) {
+				adminUsername = adminUsername + "@" + TENANT1
+			}
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.AdvancedThrottlePolicyType)
+		})
+	}
+}
+
+// Import an already existing Custom Throttling Policy to the destination env without update
+func TestImportUpdateConflictCustomThrottlePolicy(t *testing.T) {
+	for _, user := range testCaseUsers {
+		// Custom throttling policy importation
+		if isTenantUser(user.CtlUser.Username, TENANT1) {
+			continue
+		}
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, dev, user.Admin.Username, user.Admin.Password, apim.CustomThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     customThrottlePolicyFlag,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := superAdminUser
+			adminPassword := superAdminPassword
+			testutils.ValidateThrottlePolicyImportUpdateConflict(t, args, adminUsername, adminPassword, apim.CustomThrottlePolicyType)
+		})
+	}
+}
+
+// Get  Throttle Policy List APICTL output and check whether all policies are included
+func TestGetThrottlePoliciesList(t *testing.T) {
+	//devops users don't have access to view throttling policies
```

Comment:
Is this correct? If the devops users cannot view policies, this feature is wrong :-)

Reply:
fixed with d69b778e8f75fc146c5b14fb9c733afbebf60f41